dubbo-rpc/dubbo-rpc-dubbo/src/main/java/org/apache/dubbo/rpc/protocol/dubbo/DubboCodec.java

```diff
@@ -152,12 +154,31 @@ protected Object decodeBody(Channel channel, InputStream is, byte[] header) thro
                 } else {
                     req = new HeartBeatRequest(id);
                     DecodeableRpcInvocation inv;
+                    String exPs = System.getProperty(EXCEPTION_PROCESSOR_KEY);
+                    boolean existExpHandler = false;
+                    ExtensionLoader<ExceptionProcessor> extensionLoader;
+                    ExceptionProcessor expProcessor = null;
+                    if (StringUtils.isNotBlank(exPs)) {
+                        existExpHandler = true;
+                        extensionLoader = channel.getUrl().getApplicationModel().getExtensionLoader(ExceptionProcessor.class);
```

Comment:
```suggestion
                        extensionLoader = channel.getUrl().getFrameworkModel().getExtensionLoader(ExceptionProcessor.class);
```

Reply:
channel.getUrl() does not have a getApplicationModel method, use getOrDefault instead